package.json

```diff
@@ -76,13 +76,13 @@
     "ember-did-resize-modifier": "^1.0.0",
     "ember-export-application-global": "^2.0.1",
     "ember-fetch": "^8.0.2",
-    "ember-flatpickr": "^2.16.3",
+    "ember-flatpickr": "^2.16.4",
```

Comment:
This seems unrelated to the failures?

Reply:
There was another failure from a patch version of flatpickr.